models/netsuite/stg_netsuite__customers.sql

```diff
@@ -41,7 +41,7 @@ final as (
         _fivetran_deleted
 
         --The below macro adds the fields defined within your customers_pass_through_columns variable into the staging model
-        {{ fivetran_utils.fill_pass_through_columns('customers_pass_through_columns') }}
+        {{ netsuite_source.fill_pass_through_columns(var('customers_pass_through_columns')) }}
```

Comment:
does there need to be some kind of default value for these? We don't have these configurations set up and it dbt seems to not like it. example buildkite failure: https://buildkite.com/fivetran/analytics-test-pipeline/builds/26016#01944834-4a7c-4c14-80b2-85966e673cd0